service.nextup.notification/service.py

```diff
@@ -34,16 +37,32 @@ def logMsg(self, msg, lvl=1):
     def ServiceEntryPoint(self):
         player = Player()
         monitor = xbmc.Monitor()
-
+        
         lastFile = None
         lastUnwatchedFile = None
+
+        ga = GoogleAnalytics()
```

Comment:
Cool, I took it out now anyways but it is very handy especially for getting errors without the need for a user to supply a log. But of course I understand you need to be extra careful here so fully understand and that was why I called it GoogleAnalytics so that it stood out in the review :)